plugins/meta/tuning/README.md

```diff
@@ -32,3 +32,32 @@ Some network sysctls are documented in the Linux sources:
 - [Documentation/sysctl/net.txt](https://www.kernel.org/doc/Documentation/sysctl/net.txt)
 - [Documentation/networking/ip-sysctl.txt](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt)
 - [Documentation/networking/](https://www.kernel.org/doc/Documentation/networking/)
+
+## Interface Attribute Operation
+The parameters, "mac", "mtu" and "promisc", changes the interface attributes as followings:
+
+```
+{
+  "name": "mytuning",
+  "type": "tuning",
+  "promisc": true,
+  "mac": "c2:b0:57:49:47:f1",
+  "mtu": 1454
+}
+```
+
+## Interface attribute configuration reference
+
+* `mac` (string, optional): MAC address (i.e. hardware address) of interface
+* `mtu` (integer, optional): MTU of interface
+* `promisc` (bool, optional): Change the promiscas mode of interface
```

Comment:
Shouldn't that be `promiscuous`? Looks like a typo.

Reply:
Good catch! thank you so much!

Reply:
Also fixed same typo in plugins/meta/tuning/tuning_test.go.

Reply:
I see this fix tuning_test.go, but what with this one?

Reply:
Uploaded.